stginga/gingawrapper.py

```diff
@@ -51,9 +51,11 @@ def run_stginga(sys_argv):
     #     new_argv[new_argv.index('-t') + 1] = 'qt'
 
     # Auto start core global plugins
-    for gplgname in ('ChangeHistory', ):
+    for gplgname in ('ChangeHistory', 'Zoom', 'Header'):
```

Comment:
I am not sure about this because I have `wss_tools` set up to use `ginga_config.py` downstream but I am not its maintainer anymore. So the least disruptive, the better. Is this really necessary?

Over there, we use different flavors of `ginga_config.py` dependending on the "operation mode":

https://github.com/spacetelescope/wss_tools/tree/master/wss_tools/quip/config